docs/docs-content/integrations/vault.md

```diff
@@ -12,94 +12,115 @@ tags: ['packs', 'vault', 'security']
+|`charts.server.dataStorage`| Controls the size, location, storage class of the persistent storage used by the Vault. | |
 
-* **0.11.0**
+:::caution
 
-</TabItem>
+When using Vault with the RKE2 distribution of Kubernetes in Palette Edge, you must explicitly specify a storage class for the Vault server. To specify a storage class, change the value of the field  `charts.vault.server.dataStorage.storageClass` in `values.yaml` for the Vault pack in your cluster profile from `null` to a storage class that meets your needs. Refer to [Kubernetes documentation on storage classes](https://kubernetes.io/docs/concepts/storage/storage-classes/) for more details. 
 
-<TabItem label="0.9.x" value="0.9.x">
+:::
 
-  * **0.9.0**
+### Usage
 
-</TabItem>
+HashiCorp provides many uses cases for Vault. For examples, refer to [HashiCorp Vault documentation](https://developer.hashicorp.com/vault/docs/use-cases). 
 
 
-<TabItem label="0.6.x" value="0.6.x">
+#### Initialize and Unseal Vault
 
-* **0.6.0**
+If you did not configure Vault to automatically initialize in the cluster profile, you need to initialize the first root token and keys that can be used to unseal Vault.
+You can do so by following these steps:
 
-</TabItem>
+1. Log in to [Palette](https://console.spectrocloud.com).
 
-<TabItem label="0.3.x" value="0.3.x">
+2. Navigate to the left **Main Menu** and select **Clusters**.
 
-* **0.3.1**
+3. Select the cluster that has Vault installed to view its details page.
 
-</TabItem>
-</Tabs>
+4. Download the cluster **kubeconfig** file. 
 
+5. Set up your local kubectl environment to use the **kubeconfig** file you downloaded. Review the [Access Cluster with CLI](../clusters/cluster-management/palette-webctl.md) guide for additional guidance. 
 
-## Components
+6. You need to get the Vault namespace and application name. Issue the following command to get the unique values.
 
-Vault integration has the following components:
+    <br />
 
-* Vault server.
-* UI (Optional).
-* [Agent injector](https://www.vaultproject.io/docs/platform/k8s/injector/) (Optional).
+    ```shell
+    VAULT_NAMESPACE=$(kubectl get pods --selector app.kubernetes.io/name=vault --all-namespaces --output jsonpath='{.items[0].metadata.namespace}') && \
+    APP_NAME=$(echo "$VAULT_NAMESPACE" | sed 's/-ns$//')
+    ```
 
+7. Set up port forwarding by issuing the following command so you can access the Vault UI:
 
+    ```
+    kubectl port-forward $APP_NAME 8200:8200 --namespace $VAULT_NAMESPACE
+    ```
 
-## Supported Use cases
+8. Open your browser and access the [Vault UI](https://localhost:8200/ui. You will receive a warning due to using a self-signed certificate, but you can ignore this warning. Follow the prompts on the UI to initialize your root token.
```

Comment:
@lennessyy What is the root token? Can I set the root token value through the YAML? Can I get it by reviewing a kubernetes secret?

Reply:
I can add a brief definition of root tokens, but I don't think you can set root token value through the YAML unless you are using Dev mode instead of Prod mode. If you enable Dev mode, then yes, you can set it in the YAML and can probably set some other configuration to get it by reviewing Kubernetes secret, but otherwise I think you have to go through initialization:
https://developer.hashicorp.com/vault/docs/concepts/tokens#root-tokens

Reply:
I can add dev mode to the list of parameters so people know